### PR TITLE
Add new white labeling architecture article 

### PR DESCRIPTION
**Pull request by ZakKa89**

### CODE REVIEW

File: data/blog/white-labeling-a-frontend-monorepo.mdx [hunk 1]
```diff
@@ -0,0 +1,346 @@
+---
+title: "The do's and dont's with white labeling in a front-end monorepo."
+date: '2023-07-01'
+tags: ['Frontend', 'CSS', 'Monorepo', 'Design system', 'White labeling', 'Frontend Architecture']
+summary: 'Defining a good structure for your theme, component, and files is key to maintainable code and UX for multiple brands. This is the definitive guide to white labeling in a front-end monorepo.'
+authors: ['zakaria-taissate']
+theme: 'blue'
+---
+
+export const exampleImageProps = {
```
Comment (sanderdejong88) [inline data/blog/white-labeling-a-frontend-monorepo.mdx:10]: What's this about?

File: data/blog/white-labeling-a-frontend-monorepo.mdx [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+
+### 👍🏼 Do: Use global theme variables in components that have the same behaviour for all brands.
+
+Let's say you have a `PageHeader` component. This component is used by all brands. If each channel just has a different background color, we could just solve it via theming and referencing to a theme variable as a background color in the component.
+That would look something like this:
+
+#### With Styled Components
+
+```tsx
+// Location: shared/components/PageHeader/PageHeader.styles.ts
```
Comment (sanderdejong88) [inline data/blog/white-labeling-a-frontend-monorepo.mdx]: The code block allows you to add a filename. For example do it like this:
```
```tsx:shared/components/PageHeader/PageHeader.styles.ts
```

File: data/blog/white-labeling-a-frontend-monorepo.mdx (hunk 1 above)
Comment (ZakKa89) [reply data/blog/white-labeling-a-frontend-monorepo.mdx:10]: It's a way to reuse the Props I need for the images as to not to repeat myself, but this is how it's done in the mdx tool I'm using for zakka.dev, so here I should probably do something different.

File: data/blog/white-labeling-a-frontend-monorepo.mdx (hunk 2 above)
Comment (ZakKa89) [reply data/blog/white-labeling-a-frontend-monorepo.mdx]: Nice! I had no idea.

File: data/blog/white-labeling-a-frontend-monorepo.mdx (hunk 1 above)
Comment (ZakKa89) [reply data/blog/white-labeling-a-frontend-monorepo.mdx:10]: Well actually, I just checked and this seems to be the way to do it in any mdx file. 
I am using it in all of the images like this: 

image

File: data/blog/white-labeling-a-frontend-monorepo.mdx (hunk 2 above)
Comment (ZakKa89) [reply data/blog/white-labeling-a-frontend-monorepo.mdx]: Only caveat I've found is it breaks the highlighting in my editor

image

VS: 

image

**Comment by ZakKa89** [comment]
image

I changed the title a little so it fits the openGraph image. I'm still looking for a header image.

**Pull request merged**